docs/implementation.md

```diff
@@ -402,7 +402,7 @@ POST /transactions
 #### Request body schema
 ```json
 {
-  "fee": "A number representing the transaction fee to be paid to write this transaction to the blockchain.",
+  "minFee": "A number representing the minimum transaction fee to be paid to write this transaction to the blockchain. The actual fee paid can fluctuate deterministically.",
```

Comment:
How about ````minimumFree````

Reply:
Is it free though? JK, I like it, will change.